pilot/notifications.rst

```diff
@@ -0,0 +1,40 @@
+.. _notifications:
+
+Notifications
+=======================================
+
+As a creator / owner of resources inside Ronin you may receive Email based notification depending on their state.
+These intend to alert you of possible under utilization of resources to help avoid unnecessary project spend.
+
+.. note::
+    All notifications will come from the address **noreply@ronin.cloud**, always be cautious when following links from external addresses!
+
+.. _instance_utilization:
+
+Instance Utilization
+---------------------------------------
+
+If you have received an email titled "Ronin Under Utilization Alert!" this means that Ronin has noticed an :term:`instance` in your project has gone for more than more than 24 hours with under 10% CPU utilization.
+This is designed to ensure you've not accidentally left a machine running idle. Unlike on-premise VM's that run 24/7 we recommend you shut down your instances when not in use, much like you would your own PC. 
+
+We do also understand this alert could be a false positive where your workloads are not CPU demanding but still require the machine be on for extended periods.
+If this is the case please get in touch and we can make the instance exempt from these alerts.
+
+.. _unused_drive_storage:
+
+Unused Drive Storage
+---------------------------------------
+
+If you have received an email titled "Unused Drive Storage Detected" this means that Ronin has noticed detached drives have been in your project for extended periods.
+
+This could be from a terminated instance that had the "Keep On Termination" flag set:
```

Comment:
Describe what happens when this option is set. People won't necessarily understand what "keep on termination" means.

Reply:
I try to avoid adding bits like that as I feel it becomes a fine line of teaching people to suck eggs. This is where I'm disappointed that Ronin don't have their own end user documentation we can reference...

Do we really have to write our own docs on how another companies product works?
@willfurnass Maybe we add that to my list of questions for Ronin...

Reply:
People won't understand these concepts already so it's better for it to be explained simply and clearly. (It will feel too simple for us but it will be useful for them to have it spelled out.)

But I agree, the RONIN docs need to be better, it's not our job to explain their product.